### PR TITLE
Footer & Nav layout fixes

### PR DESCRIPTION
**Pull request by clhenrick**
- [x] fixes vertical alignment with footer items and page contents
- [x] updates main nav vertical alignment to be consistent with footer
- [x] updates footer logos to use b/w versions 
- [x] updates footer text
- [x] improves responsive design of Footer
- [x] renders `gtag` script in `<svelte:head>` instead of using `onMount` at runtime

### CODE REVIEW

**Comment by clhenrick** [comment]
[Site wide footer improvements](https://trello.com/c/zhwezoCR/147-site-wide-footer-improvements)

File: src/helpers/layout.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const TOP_LEVEL_WIDE_LAYOUTS = ["tools", "blog"];
+  const SUB_LEVEL_WIDE_LAYOUTS = ["help", "tools"];
+  const SUB_LEVEL_NARROW_LAYOUTS = ["blog", "events"];
+  const homepage = /^\/$/;
+  const mainPage = /^\/[a-z0-9]+(?:-[a-z0-9]+)*\/$/i;
+  const subPage = /^\/[a-z0-9]+(?:-[a-z0-9]+)*\/[a-z0-9]+(?:-[a-z0-9]+)*/i;
+
+  let isWideLayout = false;
+
+  if (homepage.exec(slug)) {
```
Comment (clhenrick) [inline src/helpers/layout.js]: Should probably use test instead of exec for these conditionals

File: src/helpers/layout.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ */
+export function hasWideLayout(slug) {
+  const TOP_LEVEL_WIDE_LAYOUTS = ["tools", "blog"];
+  const SUB_LEVEL_WIDE_LAYOUTS = ["help", "tools"];
+  const SUB_LEVEL_NARROW_LAYOUTS = ["blog", "events"];
+  const homepage = /^\/$/;
+  const mainPage = /^\/[a-z0-9]+(?:-[a-z0-9]+)*\/$/i;
+  const subPage = /^\/[a-z0-9]+(?:-[a-z0-9]+)*\/[a-z0-9]+(?:-[a-z0-9]+)*/i;
+
+  let isWideLayout = false;
```
Comment (clhenrick) [inline src/helpers/layout.js:14]: Could just return true/false instead of using a local variable

**Review by mukhtyar — APPROVED** [review]
Looks good.

**Pull request merged**